docs/tools/tikv-control.md

```diff
@@ -253,28 +252,28 @@ success!
 ```
 
 > **Note**:
-> 
+>
 > - This command only supports the local mode. It prints `success!` when successfully run.
 > - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
 > - You need to run this command for all stores where specified Regions' peers are located.
 
 ### Ldb Command
 
-The ldb command line tool offers multiple data access and database administration commands. Some examples are listed below. 
-For more information, refer to the help message displayed when running `tikv-ctl ldb` without any arguments or check the documents from RocksDB.
+The ldb command line tool offers multiple data access and database administration commands. Some examples are listed below.
+For more information, refer to the help message displayed when running `tikv-ctl ldb` or check the documents from RocksDB.
 
 Examples of data access sequence:
 
 To dump an existing RocksDB in HEX:
 
 ```bash
-$ tikv-ctl ldb --db=/tmp/test_db dump --hex > /tmp/dbdump
+$ tikv-ctl ldb --hex --db=/tmp/db dump
```

Comment:
can we still use `> /tmp/dbdump` to redirect the output?

Reply:
Yes, of course, it's just a common output redirection, not specific to `ldb`.